sanic/mixins/startup.py

```diff
@@ -385,8 +372,7 @@ def prepare(
 
         if single_process and (fast or (workers > 1) or auto_reload):
             raise RuntimeError(
-                "Single process cannot be run with multiple workers "
-                "or auto-reload"
+                "Single process cannot be run with multiple workers " "or auto-reload"
```

Comment:
Probably just consolidate to  a single string.

Reply:
Fixed.

Reply:
Fixed all these by applying Ruff rule ISC001 once. Keeping the rule disabled, however, as Ruff warns it might interfere with formatting.